source/docs/articles/sites/code.md

```diff
@@ -3,12 +3,69 @@ layout: landing
 code: true
 use:
     - code
-title: Pantheon Dashboard Code Tool
-description: Learn how to work with your site's code on Pantheon's Dashboard code tool.
+title: Your Site Code on Pantheon
+description: Learn how to work with your site's code on the Pantheon Dashboard.
 keywords: code, commit, sftp, development, how to connect, connection information, wp-admin, admin, administrator, codebase, repository, upstream
 ---
 The Code tool within the Pantheon Dashboard on any environment allows you to interact with your site's code and review the commit log.
 
+## Pantheon Git Repository
+Your codebase is required to import your site into Pantheon, as it is used to create the initial code repository.
+
+### Drupal Code Structure
+Your archive should include your entire Drupal codebase, including modules, themes, installation profiles, libraries, etc.  
+
+The code archive should not include the "files" directory (e.g. sites/default/files) or any other static assets that you do not want tracked in version control.
+
+The code archive should include the following directories:
+
+    ├── includes
+    ├── index.php
+    ├── misc
+    ├── modules
+    ├── profiles
+    ├── scripts
+    ├── sites
+        └── all
+           ├── modules
+           └── themes
+        └── default
+           └── settings.php
+    └── themes
+
+<div class="alert alert-info" role="alert">
+<strong>Note</strong>:  The <code>files</code> directory has been omitted. Remember not to include it in your codebase.</div>
```

Comment:
This notice pertains to WordPress code too, with the directory `wp-content/uploads`